Parameters:
Batch Size: 32
Tensor Size: (3,8,112,112)
Skip Length: 2
Precrop: False
Total Epochs: 1000
Learning Rate: 0.0001
Training...
	Batch 10/122 Loss:0.10761 con1:0.00177 con2:0.00177 recon1:0.05307 recon2:0.05100
Traceback (most recent call last):
  File "trainpan328.py", line 251, in <module>
    train_model()
  File "trainpan328.py", line 191, in train_model
    train(epoch)
  File "trainpan328.py", line 51, in train
    for batch_idx, (vp_diff, vid1, vid2) in enumerate(trainloader):
  File "/home/erikqu/.conda/envs/myenv/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 568, in __next__
    return self._process_next_batch(batch)
  File "/home/erikqu/.conda/envs/myenv/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 608, in _process_next_batch
    raise batch.exc_type(batch.exc_msg)
UnboundLocalError: Traceback (most recent call last):
  File "/home/erikqu/.conda/envs/myenv/lib/python3.7/site-packages/torch/utils/data/_utils/worker.py", line 99, in _worker_loop
    samples = collate_fn([dataset[i] for i in batch_indices])
  File "/home/erikqu/.conda/envs/myenv/lib/python3.7/site-packages/torch/utils/data/_utils/worker.py", line 99, in <listcomp>
    samples = collate_fn([dataset[i] for i in batch_indices])
  File "/home/yogesh/kara/REU2019/phase2/PanopticDataLoader.py", line 92, in __getitem__
    vp2 = self.get_view(seq_id=sample_name, view_id=self.view2, x_pos=pixel_index[0], y_pos=pixel_index[1])[:3]
  File "/home/yogesh/kara/REU2019/phase2/PanopticDataLoader.py", line 274, in get_view
    R = c_data["R"]
UnboundLocalError: local variable 'c_data' referenced before assignment

